### ISSUE TITLE
npm run watch:android not picking up on changes

### ISSUE THREAD

**Issue by tjvantoll** [nativescript-vue/vue-cli-template#25]
Hey all,

First of all, fantastic work on this template. I tried it out today for the first time and I’m really enjoying this workflow. I did however hit one big problem.

### Steps:

1. Run `npm run watch:android`.
2. Change a file in the project. (I’ve tried `.vue` files and `.scss` files.)

### Expected

The Android app refreshes, and the updates are in place.

### Actual

The Android app refreshes, but no updates have occurred.

### Notes

* This same workflow works fine on iOS.
* The app does refresh, meaning the watcher is for sure detecting the change.
* If I use `npm run clean` and then `npm run watch:android` again I see my changes. But I have to keep using `npm run clean` after every update to continue seeing my changes.

Any help on this would be appreciated. Thanks!

**Comment by tjvantoll** [comment]
Unfortunately it seems like this is also a problem with 0.3.0. Here’s a short video showing the problem in action: https://www.dropbox.com/s/rk784ctl94oaca2/issue.mov?dl=0.

**Comment by Crammaman** [comment]
OK so I think I've got this one figured. 

It started happening to me when I went to use an emulator with Google Play already installed (many of the premade ones on Android Studio don't have it and its required for the nativescript-google-maps-sdk to work). Turns out it was the emulators API version v28 doesn't work whereas v26 does.

tl;dr get yo'self a different emulator until Telerik figure out v28, I recommend Nexus 5 API 26
